activities/staff-management/parental-leave.md

```diff
@@ -0,0 +1,19 @@
+---
+type: Guide
+explains: how parental leave works. links to the government's website for detailed information
+---
+
+# Maternity and paternity leave
+
+Pregnant employees are entitled to 4-6 weeks pregnancy leave 
+(before the due date) and at least 10 weeks maternity leave (after childbirth). 
+
+### Paternity leave
+
+You are entitled to 1 week of partner or paternity leave after your partner gives birth.
+
+### More information
+
+[More information about maternity and partner / paternity leave can be found here:](https://business.gov.nl/regulation/leave-schemes/)
+
+If you have questions about parental leave, please reach out to HR <hr@publiccode.net>.
```

Comment:
What I'm missing here, and what should be leading is 'how do you do it'. As a guide it should set out what you need to do if you or your partner is pregnant.